docs/download.md

```diff
@@ -18,66 +18,78 @@ cuDF jar, that is either preinstalled in the Spark classpath on all nodes or sub
-need to ensure the minimum driver (450.80.02) and CUDA toolkit are installed on each Spark node.
+on V100, T4, A10, A100, L4 and H100 GPUs with CUDA 11.8-12.0.  For those using other types of GPUs 
+which do not have CUDA forward compatibility (for example, GeForce), CUDA 11.8 or later is required.
 
 ### Verify signature
-* Download the [RAPIDS Accelerator for Apache Spark 23.06.0 jar](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/23.06.0/rapids-4-spark_2.12-23.06.0.jar)
-  and [RAPIDS Accelerator for Apache Spark 23.06.0 jars.asc](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/23.06.0/rapids-4-spark_2.12-23.06.0.jar.asc)
+* Download the [RAPIDS Accelerator for Apache Spark 23.08.0 jar](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/23.08.0/rapids-4-spark_2.12-23.08.0.jar)
+  and [RAPIDS Accelerator for Apache Spark 23.08.0 jars.asc](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/23.08.0/rapids-4-spark_2.12-23.08.0.jar.asc)
 * Download the [PUB_KEY](https://keys.openpgp.org/search?q=sw-spark@nvidia.com).
 * Import the public key: `gpg --import PUB_KEY`
-* Verify the signature: `gpg --verify rapids-4-spark_2.12-23.06.0.jar.asc rapids-4-spark_2.12-23.06.0.jar`
+* Verify the signature: `gpg --verify rapids-4-spark_2.12-23.08.0.jar.asc rapids-4-spark_2.12-23.08.0.jar`
 
 The output if signature verify:
 
 	gpg: Good signature from "NVIDIA Spark (For the signature of spark-rapids release jars) <sw-spark@nvidia.com>"
 
 ### Release Notes
 New functionality and performance improvements for this release include:
-* Enhanced operator support with an OOM retry framework to minimize OOM or GPU specific config changes 
-* Spill framework to reduce OOM issues to minimize OOM or GPU specific config changes 
-* AQE for skewed broadcast hash join performance improvement
-* Support JSON to struct
-* Support StringTranslate 
-* Support windows function with string input in order by clause
-* Support regular expressions with line anchors in choice input  
-* Support rlike function with line anchor input
-* Improve the performance of ORC small file reads
+* Compatibility with Databricks AWS & Azure 12.2 ML LTS.
+* Enhanced stability and support for ORC and Parquet.
+* Reduction of out-of-memory (OOM) occurrences.
+* Corner case evaluation for data formats, operators and expressions
 * Qualification and Profiling tool:
-  * Qualification tool support for Azure Databricks
-  * The Qualification and Profiling tools do not require a live cluster, and only require read permissions on clusters
-  * Improve Profiling tool recommendations to support more tuning options
-  
+  * Profiling tool now supports Azure Databricks and AWS Databricks.
+  * Qualification tool can provide advice on unaccelerated operations.
+  * Improve user experience through CLI design.
+  * Qualification tool offers configuration and migration recommendations.
```

Comment:
@mattahrens shall we clarify that this feature is only for CPSs?

Reply:
Yes, last item is only for CSPs: `Qualification tool offers configuration and migration recommendations.`

Reply:
Is that feature available only for EMR and Dataproc for now?

Reply:
Yes, that's true -- only on EMR and Dataproc and not on Databricks (yet).